cypress/support/pages/newArticle.pageObject.js

```diff
@@ -0,0 +1,52 @@
+import PageObject from '../PageObject';
+
+class NewArticlePageObject extends PageObject {
+    url = '/#/editor';
+
+    get titleField() {
+        return cy.getByDataCy('title-new-article');
+    }
+
+    get descriptionField() {
+        return cy.getByDataCy('description-new-article');
+    }
+
+    get bodyField() {
+        return cy.getByDataCy('body-new-article');
+    }
+
+    get newArticleBtn() {
+        return cy.getByDataCy('publish-new-article')
```

Comment:
There is a missing semicolon at the end of the `return` statement in the `newArticleBtn` getter method. Consider adding a semicolon to maintain consistency and prevent potential issues.